receiver/googlecloudmonitoringreceiver/config.go

```diff
@@ -0,0 +1,59 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package googlecloudmonitoringreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/googlecloudmonitoringreceiver"
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"go.opentelemetry.io/collector/receiver/scraperhelper"
+)
+
+const minCollectionIntervalSeconds = 60
+
+type Config struct {
+	scraperhelper.ControllerConfig `mapstructure:",squash"`
+
+	Region            string    `mapstructure:"region"`
```

Comment:
What is the region going to be used for?  Are you going to call a regional endpoint, or are you using this as part of the query filter?

Reply:
Since it's optional, removing it from the config.